### PR TITLE
adds create and get endpoints

### PR DESCRIPTION
**Pull request by jafow**

### CODE REVIEW

**Comment by jafow** [comment]
the GET is unfinished but i wanted to open this now while we talking about it

File: backend/migrations/README [hunk 1]
```diff
@@ -1 +0,0 @@
-Generic single-database configuration.
```
Comment (theswerd) [inline backend/migrations/README:1]: Shouldn't we delete this file if its empty?

**Review by theswerd — COMMENTED** [review]
LGTM

**Review by theswerd — APPROVED** [review]

File: backend/migrations/README (hunk 1 above)
Comment (jafow) [reply backend/migrations/README:1]: yes will remove it's just generated file

**Pull request merged**